src/ui/components/RatingsStatsPopover/index.tsx

```diff
@@ -188,6 +232,7 @@ const RatingsStatsPopover = ({
 		>
 			<div className="mb-2">{nameBlock}</div>
 			<RatingsStats ratings={ratings} stats={stats} type={type} />
+			<PlayerNote note={note} playerName={name} />
```

Comment:
If the note exists, add some vertical spacing between stats and note - same amount as between stats and ratings, probably.